### PR TITLE
feat: Add government RP Page

### LINKED ISSUE #574: Create Government RP

### PR DESCRIPTION
**Pull request by talwinder50**
closes #574

Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

**Review by rolsonquadras — APPROVED** [review]

File: cmd/rp-rest/static/didcommvp.html [hunk 1]
```diff
@@ -230,13 +230,20 @@ <h4 class="text-xl text-gray-900 leading-tight">Presented Digital ID</h4>
                 createClaimTable(flattened)
                 document.getElementById("proceedClick").href = '/creditsuccess';
 
+            } else if (csKey === "Credit Card Statement" && csKey === "Driving License") {
+                flattened = json['_claim_sources'][key]['claims']
+                replaceKeys(flattened)
+                console.log("flat here", flattened)
```
Comment (DRK3) [inline cmd/rp-rest/static/didcommvp.html:236]: Did you intend to keep this log statement in?

**Review by DRK3 — CHANGES_REQUESTED** [review]
There's also a jpeg that looks like it's being stored directly instead of with LFS

File: cmd/rp-rest/static/didcommvp.html (hunk 1 above)
Comment (talwinder50) [reply cmd/rp-rest/static/didcommvp.html:236]: I removed it,

**Comment by talwinder50** [comment]
> There's also a jpeg that looks like it's being stored directly instead of with LFS

i added jpeg into gitattributes file.

**Review by DRK3 — APPROVED** [review]

**Pull request merged**